lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -148,6 +154,19 @@ def fake_metadata_controller(tar_path):
         cm = CacheManager(server_config, make_logger)
         # fetching metadata from metadata.log file and key/value not
         # being there should result in a MetadataError
+        tar_name = source_tarball.name.removesuffix(".tar.xz")
+
+        expected_metaerror = f"A problem occurred processing metadata.log from {source_tarball!s}: \"\'filename {tar_name}/metadata.log not found\'\""
```

Comment:
This looks odd to me:  do we really need to escape the _single_-quotes when they are inside double-quotes?